src/ResultField/GoogleShopping.php

```diff
@@ -141,7 +133,6 @@ public function generateResultFields(array $formatSettings = []):array
 				'item.manufacturer.name',
 				'item.manufacturer.externalName',
                 'item.conditionApi',
-                'texts.urlPath',
```

Comment:
Das wird oben noch benutzt.

Reply:
this modification was only related to grouping the `texts` result fields. All of them are grouped in the `$list` and added at the end of the result field array.